/home/gddaslab/rsjxw007/.local/lib/python3.10/site-packages/torch/nn/modules/loss.py:536: UserWarning: Using a target size (torch.Size([20])) that is different to the input size (torch.Size([1, 20])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
Traceback (most recent call last):
  File "/gpfs0/home1/gddaslab/rsjxw007/ABMNet/spatial_evaluate.py", line 40, in <module>
    plot_histograms(test_dataset=ground_truth,predictions=predictions, output="graphs/gnn/out")
  File "/gpfs0/home1/gddaslab/rsjxw007/ABMNet/modules/utils/graph.py", line 45, in plot_histograms
    plt.savefig(output + str(i) +'.png')
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/pyplot.py", line 977, in savefig
    res = fig.savefig(*args, **kwargs)
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/figure.py", line 3058, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 2319, in print_figure
    result = print_method(
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/backend_bases.py", line 1648, in wrapper
    return func(*args, **kwargs)
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/_api/deprecation.py", line 415, in wrapper
    return func(*inner_args, **inner_kwargs)
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py", line 541, in print_png
    mpl.image.imsave(
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/matplotlib/image.py", line 1675, in imsave
    image.save(fname, **pil_kwargs)
  File "/export/apps/opt/qiskit/0.34.2/envs/qiskit/lib/python3.10/site-packages/PIL/Image.py", line 2317, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: 'graphs/gnn/out0.png'
Job Statistics for 3762865:
           JobID       User               Start                 End    Elapsed     MaxRSS   TotalCPU      State Exit        NodeList                                  ReqTRES 
---------------- ---------- ------------------- ------------------- ---------- ---------- ---------- ---------- ---- --------------- ---------------------------------------- 
         3762865   rsjxw007 2023-05-30T10:15:23 2023-05-30T10:15:35   00:00:12             00:16.725     FAILED  1:0   r1pl-hpcf-g04 billing=16,cpu=16,gres/gpu=1,mem=119280+ 
   3762865.batch            2023-05-30T10:15:23 2023-05-30T10:15:35   00:00:12          0  00:16.722     FAILED  1:0   r1pl-hpcf-g04                                          
  3762865.extern            2023-05-30T10:15:23 2023-05-30T10:15:35   00:00:12          0  00:00.003  COMPLETED  0:0   r1pl-hpcf-g04                                          
CPU Efficiency: 8.85% of 00:03:12 core-walltime
